controllers/chat/deleteMessage.js

```diff
@@ -18,27 +18,27 @@ const deleteMessage = async (req, res) => {
     await client.partialUpdateMessage(messageID, {
       set: {
         message_type: MESSAGE_TYPE.DELETED,
-        text: 'This message has been deleted'
+        text: 'This message has been deleted',
+        attachments: []
       }
     });
-    let destroy = await client.deleteMessage(messageID);
+
     // sent empty message to channel
     let baseMessage = {
       user_id: req.userId,
-      message_type: 'deleted',
+      message_type: 'notification-deleted',
+      text: 'This message has been deleted',
       deleted_message_id: messageID,
-      deleted_message_created_at: message?.created_at,
-      deleted_message_updated_at: message?.updated_at
+      deleted_message_created_at: message?.message?.created_at,
+      deleted_message_updated_at: message?.message?.updated_at
     };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The creation of the `baseMessage` object seems redundant as it's just repeating the same information that was already set in the `partialUpdateMessage` call. It would be more efficient to create this object once and reuse it in both places.



<!-- This is an auto-generated comment by OSS CodeRabbit -->